ioc-rix-sp1k1-calc.archive

```diff
@@ -0,0 +1,4 @@
+SP1K1:MONO:CALC:ENERGY	30 monitor
+SP1K1:MONO:CALC:CFF	30 monitor
+SP1K1:MONO:CALC:BANDWIDTH	30 monitor
+SP1K1:MONO:CALC:GRATING	30 monitor
```

Comment:
Are these not on this IOC?  Since it seems like you're adding all the PVs on this IOC to the archive tile.